b2sdk/account_info/abstract.py

```diff
@@ -51,6 +51,15 @@ def clear(self):
         Remove all stored information.
         """
 
+    @abstractmethod
```

Comment:
If you do that then everyone who inherited this will get a failure when they upgrade to the new version of b2-sdk-python. The right way to do it is to provide a version of this class that does not have the method in b2sdk.v2, but has it required in b2sdk.v3. In the v2 class you can inherit v3 and define a default version of the method that returns an empty list.